packages/fe-fpm-writer/src/action/types.ts

```diff
@@ -22,12 +22,7 @@ export interface CustomAction extends CustomElement {
         enabled?: string | true;
         requiresSelection?: boolean;
         position?: Position;
-        /**
-         * If not set (i.e. undefined) then no event handler is linked. If it is set true, a new one is created and linked to the action.
-         * If an existing event handler is to be used then its id needs to be provided as string.
-         */
-        eventHandler?: string | true;
-    };
+    } & EventHandler;
```

Comment:
Not a real problem but could you please add the `EventHandler` to the extends section like you did it for the column types. I am aware that in this specific case it results in the same type but I'd like it to be used consistently.

Reply:
correction submitted - https://github.com/SAP/open-ux-tools/pull/536/commits/93a705e1446ff30d218d7d563ea717d7b1c1b4ce

In result I have:
```
export interface CustomActionSettings extends EventHandler {
   ...
}

export interface CustomAction extends CustomElement {
    target: CustomActionTarget;
    settings: CustomActionSettings;
}
```

If we compare with other extension, then for other extensions interface is:
```
export interface CustomTableColumn extends CustomElement, EventHandler
```

`eventHandler` property was placed in `settings` for custom action also before my changes - [action types](https://github.com/SAP/open-ux-tools/blob/main/packages/fe-fpm-writer/src/action/types.ts#L29)
In result I did not changed placement of `eventHandler` and left it in `settings`.

I also can move it to root level to have it same as for other extensions, like:
```
export interface CustomAction extends CustomElement, EventHandler {
```